docs/documentation/cert-manager.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Cert-Manager
+description: Cert-Manager
+layout: docs
+permalink: /docs/cert-manager
+---
+### Cert-Manager
+
+Dekorate supports to generate a X.509 certificate with the help of the Certificate and Issuer CRD resources handled by the [Cert-Manager](https://cert-manager.io/). When these CRD resources are deployed on the cluster, the Cert-Manager will process them in order to populate a Secret containing by example a: CA certificate, private key, server certificate or java keystores, etc.
+
+To let Dekorate to generate the certificate and issuer resources, simply declare the following dependency part of your pom file:
+
+```xml
+<dependency>
+  <groupId>io.dekorate</groupId>
+  <artifactId>certmanager-annotations</artifactId>
+  <version>{{site.data.project.release.current-version}}</version>
+</dependency>
+```
+
+And provide the certificate configuration. The minimal information that the Dekorate needs is:
+- `secretName` : the name of the Kubernetes [Secret](https://kubernetes.io/docs/concepts/configuration/secret/) resource that will include the Cert-Manager generated files.
+- the Issuer that represents certificate authorities (CAs). See all the supported options in [the Issuer](#issuers) section.
```

Comment:
`the Issuer that represents certificate authorities (CAs).` --> `the Issuer that represents the certificate authorities (CAs).` Question: Should we use the plural or singular form here (CA vs CAs) ? As I think that we are just here generating a CA (= selfsigned CA) which is not trusted by some root certificate authorities, then the singular form should be enough